recipes/viralflye/meta.yaml

```diff
@@ -0,0 +1,51 @@
+---
+{% set version = "0.2" %}
+
+package:
+  name: viralflye
+  version: {{ version }}
+
+source:
+  url: https://github.com/Dmitry-Antipov/viralFlye/archive/v{{ version }}.tar.gz
+  sha256: bf7c928628e6fd5c34e6112c40c2fceece62f920acb384838b5e38dd7544af08
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - git
+  host:
+    - python ==3.10.8
+  run:
+    - bcftools ==1.16
+    - biopython ==1.81
+    - blast ==2.13.0
+    - bwa ==0.7.17
+    - freebayes ==1.3.6
+    - minced ==0.4.2
+    - minimap2 ==2.24
+    - numpy ==1.24.2
+    - prodigal ==2.6.3
+    - pysam ==0.20.0
+    - samtools ==1.16.1
+    - scipy ==1.10.1
+    - seqtk ==1.3
+    - tabix ==1.11
+    - vcflib ==1.0.3
+    - viralverify ==1.1
+    # - viralcomplete NO TAGGED RELEASE
+
+test:
+  commands:
+    # both exit 1
+    - viralcomplete -h &> /dev/null || [[ "$?" == 1 ]]
+    - viralFlye.py  2> /dev/null || [[ "$?" == 1 ]]
```

Comment:
can you grep for expected output here?

Reply:
Fixed https://github.com/bioconda/bioconda-recipes/pull/39987/commits/99ab3963c2e1d84274f1d7a96b3cf280fe938f51